### PR TITLE
add additional addresses frontend

### PR DESCRIPTION
**Pull request by patrickweh**

### CODE REVIEW

File: src/Livewire/Order/AdditionalAddresses.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    {
+        $this->form->addresses = array_filter(
+            $this->form->addresses,
+            fn (array $address) => $address['address_id'] !== $id
+        );
+
+        try {
+            $this->form->save();
+        } catch (ValidationException $e) {
+            exception_to_notifications($e, $this);
```
Comment (SirSplasch) [inline src/Livewire/Order/AdditionalAddresses.php:60]: revert removal of given address_id if validation failed.

File: src/Livewire/Order/AdditionalAddresses.php
Comment (SirSplasch) [inline src/Livewire/Order/AdditionalAddresses.php:1]: why not catch UnauthorziedException?

File: src/Livewire/Order/AdditionalAddresses.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    public function render(): View
+    {
+        $this->form->id = $this->orderId;
+        $this->form->addresses = resolve_static(\FluxErp\Models\Order::class, 'query')
+            ->whereKey($this->orderId)
+            ->with('addresses')
+            ->first(['id'])
+            ->addresses
+            ->map(fn ($address) => [
+                'address_type' => $address->pivot->addressType->name,
```
Comment (SirSplasch) [inline src/Livewire/Order/AdditionalAddresses.php:34]: why duplicate address_type and type?

File: resources/views/livewire/order/additional-addresses.blade.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                <x-card>
+                    <div class="flex flex-col gap-4">
+                        <x-select
+                            :label="__('Address')"
+                            option-value="id"
+                            option-label="label"
+                            template="user-option"
+                            wire:model="address_id"
+                            :async-data="[
+                                'api' => route('search', \FluxErp\Models\Address::class),
```
Comment (SirSplasch) [inline resources/views/livewire/order/additional-addresses.blade.php:14]: can this be any address? Not constraint by contact_id or client_id?

File: resources/views/livewire/order/additional-addresses.blade.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                                  ],
+                                  'with' => 'contact.media',
+                                ]
+                            ]"
+                        />
+                        <x-select
+                            :label="__('Type')"
+                            option-key-value
+                            wire:model="address_type_id"
+                            :options="resolve_static(\FluxErp\Models\AddressType::class, 'query')->pluck('name', 'id')"
```
Comment (SirSplasch) [inline resources/views/livewire/order/additional-addresses.blade.php:29]: remove address types from differing clients?

**Review by SirSplasch — CHANGES_REQUESTED** [review]

File: src/Livewire/Order/AdditionalAddresses.php (hunk 1 above)
Comment (patrickweh) [reply src/Livewire/Order/AdditionalAddresses.php:60]: happens in line 78

File: resources/views/livewire/order/additional-addresses.blade.php (hunk 3 above)
Comment (patrickweh) [reply resources/views/livewire/order/additional-addresses.blade.php:14]: can be any address, yes

File: resources/views/livewire/settings/plugins.blade.php [hunk 5]
```diff
@@ -47,13 +47,13 @@
         <x-modal name="install" max-width="7xl">
             <x-card :title="__('Install packages')">
                 <div class="flex flex-col gap-4">
-                    <x-features.media.upload-form-object wire:model="file" :multiple="true" accept=".zip,.rar,.7zip">
+                    <x-flux::features.media.upload-form-object wire:model="file" :multiple="true" accept=".zip,.rar,.7zip">
                         <x-slot:footer>
                             <div x-show="$wire.file.stagedFiles.length > 0" x-cloak x-transition class="flex justify-end">
-                                <x-button primary spinner="installUploaded" :label="__('Upload package')" wire:click="installUploaded" wire:confirm.icon.warning="{{ __('wire:confirm.install-uploaded-plugin') }}"/>
+                                <x-button primary spinner="installUploaded" :label="__('Upload package')" wire:click="installUploaded" wire:flux-confirm.icon.warning="{{ __('wire:confirm.install-uploaded-plugin') }}"/>
```
Comment (SirSplasch) [inline resources/views/livewire/settings/plugins.blade.php:53]: remove "flux-". If it is correct to keep it, this must be changed in all other confirm dialogs.

File: src/Livewire/Accounting/DirectDebit.php [hunk 6]
```diff
@@ -34,7 +34,7 @@ protected function getSelectedActions(): array
                 ->label(__('Create Payment Run'))
                 ->attributes([
                     'wire:click' => 'createPaymentRun',
-                    'wire:confirm' => __('Create Payment Run|Do you really want to create the Payment Run?|Cancel|Yes'),
+                    'wire:flux-confirm' => __('Create Payment Run|Do you really want to create the Payment Run?|Cancel|Yes'),
```
[the remark this answers is not in the document]
Comment (patrickweh) [reply src/Livewire/Accounting/DirectDebit.php:37]: its correct but i cant find any other wire:confirm only the translation keys which can stay as it is

**Review by SirSplasch — APPROVED** [review]

**Pull request merged**